### ISSUE TITLE
[BUG]sharing is broken in app 1.79.0

### ISSUE THREAD

**Issue by Stooovie** [immich-app/immich#4187]
### The bug

Tapping "Share" only shows a "Preparing" popup for a split seconds, and then nothing happens. 

### The OS that Immich Server is running on

Proxmox 8.0.4

### Version of Immich Server

1.79.1

### Version of Immich Mobile App

1.79.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:release
    entrypoint: ["/bin/sh", "./start-server.sh"]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
    env_file:
      - .env
    depends_on:
      - redis
      - database
      - typesense
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:release
    entrypoint: ["/bin/sh", "./start-microservices.sh"]
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
    env_file:
      - .env
    depends_on:
      - redis
      - database
      - typesense
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    image: ghcr.io/immich-app/immich-machine-learning:release
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - model-cache:/cache
    env_file:
      - .env
    restart: always 
	
 immich-web:
    container_name: immich_web
    image: ghcr.io/immich-app/immich-web:release
    entrypoint: ["/bin/sh", "./entrypoint.sh"]
    env_file:
      - .env
    restart: always

  typesense:
    container_name: immich_typesense
    image: typesense/typesense:0.24.0
    environment:
      - TYPESENSE_API_KEY=${TYPESENSE_API_KEY}
      - TYPESENSE_DATA_DIR=/data
    logging:
      driver: none
    volumes:
      - tsdata:/data
    restart: always
 
  redis:
    container_name: immich_redis
    image: redis:6.2
    restart: always

  database:
    container_name: immich_postgres
    image: postgres:14
    env_file:
      - .env
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      PG_DATA: /var/lib/postgresql/data
    volumes:
      - pgdata:/var/lib/postgresql/data
    restart: always

  immich-proxy:
    container_name: immich_proxy
    image: ghcr.io/immich-app/immich-proxy:release
    environment:
      # Make sure these values get passed through from the env file
      - IMMICH_SERVER_URL
      - IMMICH_WEB_URL
    ports:
      - 2283:8080
    logging:
      driver: none
    depends_on:
      - immich-server
    restart: always

volumes:
  pgdata:
  model-cache:
  tsdata:
```

### Your .env content

```Shell
DB_HOSTNAME=immich_postgres
DB_USERNAME=postgres
DB_PASSWORD=postgres
DB_DATABASE_NAME=immich
REDIS_HOSTNAME=immich_redis
UPLOAD_LOCATION=/root/media
TYPESENSE_API_KEY=some-random-text
PUBLIC_LOGIN_PAGE_MESSAGE=
IMMICH_WEB_URL=http://immich-web:3000
IMMICH_SERVER_URL=http://immich-server:3001
IMMICH_MACHINE_LEARNING_URL=http://immich-machine-learning:3003
```

### Reproduction steps

```bash
1. Share any asset 
2.
3.
...
```

### Additional information
Video:
[https://youtu.be/fiA0yE26IWk?si=Sg1_VeUKrmeEH7Af](url)

_No response_

**Comment by gianni-rosato** [comment]
This is happening to me as well. Using iOS

**Comment by jianglai** [comment]
Same here. On both iOS and iPadOS.

**Comment by shenlong-tanwen** [comment]
@Stooovie / @jianglai Do you guys still face this issue with the latest version of the app (1.83.0)?

**Comment by Stooovie** [comment]
No! Fixed for me :)

**Comment by jianglai** [comment]
I'm still seeing this. I think this is the corresponding log:

```
Catch all error: PlatformException(error, sharePositionOrigin: argument must be set, {{0, 0}, {0, 0}} must be non-zero and within coordinate space of source view: {{0, 0}, {1194, 834}}, null, null) - PlatformException(error, sharePositionOrigin: argument must be set, {{0, 0}, {0, 0}} must be non-zero and within coordinate space of source view: {{0, 0}, {1194, 834}}, null, null)

#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:652)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:310)
<asynchronous suspension>
#2      MethodChannelShare.shareFilesWithResult (package:share_plus_platform_interface/method_channel/method_channel_share.dart:140)
<asynchronous suspension>
```

**Comment by dodg3r** [comment]
Hi.
In version 1.86 the "preparing" popup doesnt go away.
The function works and my picture does get uploaded but preparing wont go away. I have to restart the app.

I dont find any errors in the log but maybe I´m looking at the wrong place

**Comment by Stooovie** [comment]
Yep, bug is back. 1.86.0

**Comment by mertalev** [comment]
Can you confirm if this issue is still relevant?

**Comment by Stooovie** [comment]
The issue is fixed as of app 1.98.1 and probably longer. Thanks!
